src/test/java/com/patrickjones/services/TransferManagerTest.java

```diff
@@ -0,0 +1,32 @@
+package com.patrickjones.services;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.patrickjones.models.BancAccount;
+
+public class TransferManagerTest {
+
+	@Test
+	public void testDebitsAccountToDebitByTransferAmount() {
+		BancAccount acctToDebit = new BancAccount();
+		acctToDebit.setBalance(1000);
+
+		BancAccount acctToCredit = new BancAccount();
+		acctToCredit.setBalance(700);
+
+		TransferManager transferManager = new TransferManager();
+		transferManager.transfer(acctToDebit, acctToCredit, 100);
+
+		String failureMessage = "The account to debit's balance should be reduced by the amount to transfer";
+		assertEquals(failureMessage, 900, acctToDebit.getBalance());
```

Comment:
Most of the time you will use JUnit's `Assert` classes to test your code. In this case, we're testing that the `accountToDebit`'s balance gets reduced by `100` since that is what we're transferring. 

Notice the format of the test -we set up some dummy `BancAccount` classes, set up the class we want to test (the `TransferManager` in this case), then call the method that does the action we want to test (`transfer`) in this case. After we call `transfer` we perform the assertions to test what we want to ensure either happened or didn't happen.